rfcs/InputUnion.md

```diff
@@ -327,11 +339,7 @@ type Mutation {
 }
 ```
 
-##### Problems:
-
-* The discriminator field is non-sensical if the input is used _outside_ of an input union, or in multiple input unions.
```

Comment:
Looks like this PR is removing all the `Problems` sections. I'd prefer we didn't delete them now, so that we can incorporate them into the evaluations...

Reply:
I included them into the evaluation.